ttl/client/notification.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import (
+	"context"
+
+	"github.com/pingcap/tidb/ddl/util"
+	clientv3 "go.etcd.io/etcd/client/v3"
+)
+
+const ttlNotificationPrefix string = "/tidb/ttl/notification/"
+
+// NotificationClient is a client to notify other TTL workers
+type NotificationClient interface {
```

Comment:
Is it better to merge interface `NotificationClient` and `CommandClient`?

Reply:
I don't think it's good to merge these two interfaces. They are actually doing different things, and the requirement of `CommandClient` is much higher than the `NotificationClient` (if the command somehow disappeared, the sending one will be blocked).

Also we just want to use the `CommandClient` in the test, but the `NotificationClient` is used in production codes (as we don't expect it to be 100% transfered, and the message is not important).